.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: 'ci'
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  ci:
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: 'actions/checkout@v2'
+        with:
+          fetch-depth: 0
+
+      - uses: 'wagoid/commitlint-github-action@v3'
+        with:
+          configFile: 'commitlint.config.js'
+
+      - name: 'Use Node.js'
+        uses: 'actions/setup-node@v2.2.0'
+        with:
+          node-version: 'lts/*'
```

Comment:
Does this mean there will be a matrix of all the Node.js versions?

Reply:
I would prefer GitHub Actions but while Travis currently runs in all supported Node.js versions, this seems to use only the latest LTS. Is there a nice way to make this run all supported Node.js releases without needing to update those versions manually?

Reply:
You are right, it is only using the latest LTS version, I thought otherwise.
I opened a PR to `actions/setup-node`, to support the `current` alias : https://github.com/actions/setup-node/pull/279

Reply:
@Divlo would a `current` feature in that action allow us to generate a matrix of several Node.js versions? That is what Travis CI has been doing for us.

Reply:
Yes, it will allow us to use `node-version: [lts/*, current]`, so it will run the CI for the latest LTS Node.js version (currently v14) and latest Current Node.js version (currently v16).

Reply:
To generate a matrix, a strategy like this one would have to be used: https://github.com/es-shims/Promise.prototype.finally/blob/0bbf745cf4b427aee96262a59bf369b268d6995d/.github/workflows/node-native.yml#L6-L25

Else `[10, 12, lts/*, current]` will leave gaps when a new major lts or current release is made.

Reply:
Reference: https://github.com/actions/setup-node/issues/26#issuecomment-773769435

Reply:
So assuming a matrix is supported, I'd like an additional step that reads the Node.js versions from a file in the repository that can be updated via Renovate. It would be rather hacky, but I wouldn't mind if this file is the `.travis.yml` even though we wouldn't be using Travis CI. Of course, a comment in that file explaining its existence would be in order.

Reply:
Yes, matrix is supported.

> I'd like an additional step that reads the Node.js versions from a file in the repository that can be updated via Renovate

I'm not against this, I guess you read that from https://github.com/renovatebot/renovate/issues/7716.
But I would rather have `node-version: [lts/*, current]` if my PR get merged in `actions/setup-node`.

> I wouldn't mind if this file is the `.travis.yml` even though we wouldn't be using Travis CI.

I don't think that's a good idea, we should not use a `.travis.yml` file if we are not using Travis CI.

We will see what we can do, but first we should make sure that the CI pass again and currently it is not the case (see: https://github.com/standard/eslint-config-standard-with-typescript/pull/597).